packages/monaco/src/browser/style/index.css

```diff
@@ -188,16 +169,25 @@
     font-family: var(--theia-ui-font-family);
     font-size: var(--theia-ui-font-size1) !important;
     color: var(--theia-foreground) !important;
-    overflow: hidden;
-    text-overflow: ellipsis;
 }
 
-.quick-input-list .monaco-icon-label::before {
-  transform: scale(0.8);
+.quick-input-list .monaco-icon-label.codicon,
+.quick-input-list .monaco-icon-label.file-icon {
+  display: flex;
+  text-align: left;
+}
+
+.codicon-file.default-file-icon.file-icon {
+  padding-left: 2px;
+  height: 22px;
 }
 
 .quick-input-list .monaco-icon-label.codicon::before {
-  padding-top: 3px;
+  position: relative;
+  top: 3px;
+  margin-right: 4px;
+  text-align: center;
+  font-size: var(--theia-content-font-size);
```

Comment:
As a minor point, I think the check marks look better integrated (more like the file icons) with these changes, but I've only tested under limited conditions (resolution, theme, etc.), so there may be something I'm missing.

```suggestion
  position: relative;
  top: 3px;
  text-align: center;
```

Reply:
FWIW this PR doesn't (yet) handle the part about text alignment & image size for the codicon:
![image](https://user-images.githubusercontent.com/20653241/223764729-2389be94-4a21-4acf-bb40-7dd5c6a77ac3.png)